src/etl/extract.py

```diff
@@ -0,0 +1,85 @@
+import polars as pl
+from glob import glob
+from os import path
+from logging import Logger
+
+current_file_dir = path.dirname(path.realpath(__file__))
+
+
+class Extract:
+    def __init__(self, config: dict, logger: Logger) -> None:
+        self.logger = logger
+        self.root_folder = path.dirname(path.dirname(current_file_dir))
+        self.data_file_source = config["data_file_source"]
+        self.data_file_output = config["data_file_output"]
+        self.data_columns = config["data_columns"]
+
+    def load_file(self) -> pl.DataFrame:
+        """
+        Loads data from the directory path.
+        It can handle multiple files as long they are in the same format
+        and specified on the configuration file.
+        """
+
+        # To handle other file formats, specify it in the configuration file
+        if self.data_file_source["file_extension"] == ".csv":
+            file_path = path.join(
```

Comment:
I have a slight preference to use Pathlib where appropriate: https://docs.python.org/3/library/pathlib.html